fixit/rules/use_assert_is_not_none.py

```diff
@@ -76,173 +86,86 @@ class UseAssertIsNotNoneRule(CstLintRule):
+                                            )
+                                        ]
+                                    ),
+                                ),
                             ),
+                            "argument",
                         )
                     )
                 ],
             ),
-        ):
+        )
 
-            new_call = node.with_changes(
-                func=cst.Attribute(
-                    value=cst.Name("self"), attr=cst.Name("assertIsNotNone")
-                ),
-                args=[
-                    cst.Arg(
-                        ensure_type(
-                            ensure_type(
-                                node.args[0].value, cst.UnaryOperation
-                            ).expression,
-                            cst.Comparison,
-                        ).left
-                    )
-                ],
-            )
-            self.report(node, replacement=new_call)
-        # `self.assertFalse(x is None)` -> `self.assertIsNotNone(x)`
-        elif m.matches(
-            node,
-            m.Call(
-                func=m.Attribute(value=m.Name("self"), attr=m.Name("assertFalse")),
-                args=[
-                    m.Arg(
-                        m.Comparison(
-                            comparisons=[
-                                m.ComparisonTarget(m.Is(), comparator=m.Name("None"))
-                            ]
-                        )
-                    )
-                ],
-            ),
-        ):
-            new_call = node.with_changes(
-                func=cst.Attribute(
-                    value=cst.Name("self"), attr=cst.Name("assertIsNotNone")
-                ),
-                args=[cst.Arg(ensure_type(node.args[0].value, cst.Comparison).left)],
-            )
-            self.report(node, replacement=new_call)
-        # `self.assertTrue(x is None)` -> `self.assertIsNotNone(x))
-        elif m.matches(
-            node,
-            m.Call(
-                func=m.Attribute(value=m.Name("self"), attr=m.Name("assertTrue")),
-                args=[
-                    m.Arg(
-                        m.Comparison(
-                            comparisons=[
-                                m.ComparisonTarget(m.Is(), comparator=m.Name("None"))
-                            ]
-                        )
-                    )
-                ],
-            ),
-        ):
-            new_call = node.with_changes(
-                func=cst.Attribute(
-                    value=cst.Name("self"), attr=cst.Name("assertIsNone")
-                ),
-                args=[cst.Arg(ensure_type(node.args[0].value, cst.Comparison).left)],
-            )
-            self.report(node, replacement=new_call)
+        if result:
+            assertion_name = result["assertion_name"]
+            if isinstance(assertion_name, Sequence):
+                assertion_name = assertion_name[0]
```

Comment:
Based on the description of `extract`, the value in the result dict can be either a node or a sequence. So use isinstance to teach Pyre is the right thing to do.
https://libcst.readthedocs.io/en/latest/matchers.html#libcst.matchers.extract